gourmet/plugins/import_export/web_import_plugin/generic_web_importer_plugin.py

```diff
@@ -35,15 +38,21 @@ def test_url (self, url, data, content_type):
             if 'html' in content_type:
                 return -1 # We are the fallback option
 
-    def get_web_importer (self, url, data, content_type):
+    def get_web_importer (self,
+                          url: str,
+                          data: Union[bytes, str],
+                          content_type: str):
+        # FIXME: data needn't be bytes
+        if isinstance(data, bytes):
```

Comment:
As it's always valid unicode from the webpages, do the decoding of the `reader.data` in [`importManager.finish_web_import`](https://github.com/kirienko/gourmet/blob/9176f56aaa86cde607ae6573a42db4be995d9d7a/gourmet/importers/importManager.py#L116).